documentation/docs/troubleshooting.md

```diff
@@ -7,16 +9,15 @@ keywords:
 - Discord
 - FAQ
 - Difficulties
-description: Resources to troubleshoot your node.  
-image: /img/logo/HornetLogo.png
+- Reference
 ---
 
 # Troubleshooting
 
-Check our [Frequently asked questions](faq.md).
+Check out our [Frequently asked questions](faq.md).
 
-If you can't find your question in the FAQ, feel free to ask in the `#hornet` channel in the ([official iota discord server](https://discord.iota.org/)).
+If you can not find your question in the FAQ, feel free to ask in the `#hornet` channel in the ([official iota discord server](https://discord.iota.org/)).
 
-## Something went wrong?
+## Something Went Wrong?
```

Comment:
is this supposed to be a question?

Reply:
Asking to the reader, I think. Just a more personal thing; I'd leave it tbh.

Reply:
I don't mind it to be a question, but then it must be in a proper English.

Reply:
👌